src/components/CreateEvent/CreateEvent.tsx

```diff
@@ -0,0 +1,121 @@
+        },
+        participants: [{
+          id: "5f6b242580090096cfb631a1",
+          firstName: firstName,
+          lastName: lastName,
+          email: email
+        }]
+      })
+    })
+    .then(() => {
+      console.log('Event created');
+    })
+    .catch(error => { console.log(error)})
+  }
+
+  function isFormFilled () {
+    if (name !== "" && description !== "" && eventType !== "") {
+      return true
+    } else {
+      return false
+    }
+  }
+
+  return (
+    <CreateEventWrapper>
+      <h3>Create new event</h3>
+      <Form>
+        <label htmlFor="name">
+          Event name
+          <input name="name" type="text" value={name} onChange={(event) => setName(event.target.value)}/>
+        </label>
+
+        <label htmlFor="time">
+          Date and time
+          <input name="time" type="date" value={format(parseISO(time), 'yyyy-MM-dd')} onChange={(event) => setTime(formatISO(Date.parse(event.target.value + ' ' + format(parseISO(time), 'HH:mm'))))}/>
+          <input name="time" type="time" value={format(parseISO(time), 'HH:mm')} onChange={(event) => setTime(formatISO(Date.parse(format(parseISO(time), 'yyyy-MM-dd') + ' ' + event.target.value)))}/>
+        </label>
+
+        <label htmlFor="description">
+          Description
+          <textarea name="description" value={description} onChange={(event) => setDescription(event.target.value)}></textarea>
+        </label>
+      
+        <label htmlFor="image">
+          Image link
+          <input name="image" type="text" value={image} onChange={(event) => setImage(event.target.value)}/>
+        </label>
+
+        <label htmlFor="imageAlt">
+          Image text
+          <input name="imageAlt" type="text" value={imageAlt} onChange={(event) => setImageAlt(event.target.value)}/>
+        </label>
+
+        <label htmlFor="eventType">
+          Event type
+          <select name="eventType" value={eventType} onChange={(event) => setEventType(event.target.value)}>
+            <option value="" disabled={eventType !== ""}>Choose event type</option>
+            <option value="Subject matter event">Subject matter event</option>
+            <option value="Coding event">Coding event</option>
+            <option value="Project presentation">Project presentation</option>
+            <option value="Workshop">Workshop</option>
+          </select>
+        </label>
+
+        <label htmlFor="firstName">
+          Owner first name
+          <input name="firstName" type="text" value={firstName} onChange={(event) => setFirstName(event.target.value)}/>
+        </label>
+
+        <label htmlFor="lastName">
+          Owner last name
+          <input name="lastName" type="text" value={lastName} onChange={(event) => setLastName(event.target.value)}/>
+        </label>
+
+        <label htmlFor="email">
+          Owner email
+          <input name="email" type="text" value={email} onChange={(event) => setEmail(event.target.value)}/>
+        </label>
```

Comment:
A pattern that typically repeats itself when we're writing web forms are these label-input pairs. In react, we typically create field components to avoid repeating ourselves as much. E.g.
```
function TextField({ id, name, label, value, onChange }) {
  return (
    <>
      <label htmlFor={id}>{label}</label>
      <input id={id} name={name} type="text" value={value} onChange={onChange} />
    </>
  );
}
```